### PR TITLE
remove Munch and demunch from exported names

### PR DESCRIPTION
**Pull request by andre-merzky**
remove Munch from RU namespace to force stack consistency.

### CODE REVIEW

**Comment by mtitov** [comment]
@andre-merzky should class `Description` be removed as well?

File: src/radical/utils/typeddict.py [hunk 1]
```diff
@@ -107,7 +108,7 @@ class TypedDict(dict, metaclass=TypedDictMeta):
     # --------------------------------------------------------------------------
     #
     def __init__(self, from_dict=None):
-        """
+        '''
```
Comment (mtitov) [inline src/radical/utils/typeddict.py:111]: hmm, as I remember we agreed to have `"""`

p.s. and within the class it is 1 line between methods

Comment (andre-merzky) [reply src/radical/utils/typeddict.py:111]: Really?  But we do single quotes and double lines throughout the code base otherwise...

**Comment by andre-merzky** [comment]
> @andre-merzky should class `Description` be removed as well?

Yeah, I considered that also - would be open to that for simplicity.  What do you think?
  
> **update** and remove method `demunch` that was for backward compatibility?

`demunch`: I checked the codes we have and should have catched all places where those where used (famous last words :-P - but tests should confirm).

`update`: not sure I got that: `update` is not touched by this PR?

**Comment by mtitov** [comment]
> > @andre-merzky should class `Description` be removed as well?

> Yeah, I considered that also - would be open to that for simplicity.  What do you think?

yes, I'm also up for removing it (with this it will be much clear in RP)
and up to remove `demunch` from `TypedDict` (which points to `as_dict` -> `demunch = as_dict`)

> not sure I got that: update is not touched by this PR?

sorry :) this one was as "update" I made to the post, it is not about method `update`

**Comment by andre-merzky** [comment]
> yes, I'm also up for removing it (with this it will be much clear in RP) and up to remove `demunch` from `TypedDict` (which points to `as_dict` -> `demunch = as_dict`)
> 
> > not sure I got that: update is not touched by this PR?

Good, will do so then!
 
> sorry :) this one was as "update" I made to the post, it is not about method `update`

Ah, got it :-D

**Comment by andre-merzky** [comment]
Tests fail because the prof test pulls RP devel which does not yet handle the missing `Munch`.  That should settle once the respective PR lands in RP devel.  But that PR (radical-cybertools/radical.pilot/pull/2528) waits for this one to get merged to get `as_dict` back.  So we are running in circles a bit.  I would propose to merge this RU PR first once you approve and let the changes trickle up.

**Comment by andre-merzky** [comment]
> BTW looks like tests here and in RP will fail, due to crossref...

Ah, our messages crossed, realized the same ... :)

**Pull request merged**